src/mme-app/actionHandlers/ueInitDetachActionHandlers.cpp

```diff
@@ -280,3 +280,40 @@ ActStatus ActionHandlers::handle_s1_rel_req_during_detach(ControlBlock& cb)
     return ActStatus::ABORT;
 }
 
+/***************************************
+* Action handler : handle_dsr_during_detach
+***************************************/
+ActStatus ActionHandlers::handle_dsr_during_detach(ControlBlock& cb)
```

Comment:
We would need handling of DSR in all states. do we plan to add that as another PR ??

Reply:
Currently, we've handled DSR in service request, UE and Network Initiated Detach. We shall handle other states in another PR.